src/mocks/handlers/couponsHandlers.ts

```diff
@@ -0,0 +1,29 @@
+import { rest } from "msw";
+import { Coupon } from "../../types/domain";
+
+const couponListMockData: Coupon[] = [
+  {
+    id: 1,
+    name: "반짝할인(10%)",
+    minOrderPrice: 10000,
+    maxDiscountPrice: 5000,
+    isAvailable: true,
+    discountPrice: 3000,
+    expiredAt: "2023-05-31T14:30:45.123",
+  },
+  {
+    id: 3,
+    name: "반짝할인(20%)",
+    minOrderPrice: 20000,
```

Comment:
소소한 부분이지만 Numeric separators를 사용해도 좋겠네요

```suggestion
    minOrderPrice: 20_000,
```